.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Be careful not to include your IDE config into version checking (maybe this should be added to .gitignore?)